Zilon.Core/Zilon.Core/PersonModules/HumanSurvivalModule.cs

```diff
@@ -208,5 +289,261 @@ public override void ResetStats()
+                            break;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Помещает в список бонус на ХП.
+        /// </summary>
+        /// <param name="level"> Уровень бонуса. </param>
+        /// <param name="direction"> Направление бонуса. </param>
+        /// <param name="bonuses"> Аккумулирующий список бонусов.
+        /// Отмечен ref, чтобы показать, что изменяется внутри метода. </param>
+        private void BonusToHealth(PersonRuleLevel level, PersonRuleDirection direction,
+            ref List<SurvivalStatBonus> bonuses)
+        {
+            var survivalModule = this;
+            const SurvivalStatType hpStatType = SurvivalStatType.Health;
+            var hpStat = survivalModule.Stats.SingleOrDefault(x => x.Type == hpStatType);
+            if (hpStat != null)
+            {
+                var bonus = 0;
+                switch (level)
+                {
+                    case PersonRuleLevel.Lesser:
+                        bonus = 1;
+                        break;
+
+                    case PersonRuleLevel.Normal:
+                        bonus = 3;
+                        break;
+
+                    case PersonRuleLevel.Grand:
+                        bonus = 5;
+                        break;
+
+                    case PersonRuleLevel.Absolute:
+                        bonus = 10;
+                        break;
+                }
+
+                if (direction == PersonRuleDirection.Negative)
+                {
+                    bonus *= -1;
+                }
+
+                var currentBonus = bonuses.SingleOrDefault(x => x.SurvivalStatType == hpStatType);
+                if (currentBonus == null)
+                {
+                    currentBonus = new SurvivalStatBonus(hpStatType);
+                    bonuses.Add(currentBonus);
+                }
+                currentBonus.ValueBonus += bonus;
+            }
+        }
+
+        private static void BonusToDownPass(
+            SurvivalStatType statType,
+            PersonRuleLevel level,
+            PersonRuleDirection direction,
+            ref List<SurvivalStatBonus> bonuses)
+        {
+            var currentBonusValue = 0;
+            var directionQuaff = direction == PersonRuleDirection.Negative ? -1 : 1;
+
+            switch (level)
+            {
+                case PersonRuleLevel.Lesser:
+                    currentBonusValue = 1 * directionQuaff;
+                    break;
+
+                case PersonRuleLevel.Normal:
+                    currentBonusValue = 2 * directionQuaff;
+                    break;
+
+                case PersonRuleLevel.Grand:
+                    currentBonusValue = 5 * directionQuaff;
+                    break;
+
+                case PersonRuleLevel.Absolute:
+                    currentBonusValue = 10 * directionQuaff;
+                    break;
+
+                case PersonRuleLevel.None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'case' clause.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5480188)